docs/core/diagnostics/dotnet-trace.md

```diff
@@ -6,7 +6,7 @@ ms.topic: reference
 ---
 # dotnet-trace performance analysis utility
 
-**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
+**This article applies to:** ✔️ dotnet-trace version 3 and later versions
```

Comment:
```suggestion
**This article applies to:** ✔️ dotnet-trace 3.0 and later versions
```

Reply:
This one should say the earliest version of dotnet-trace, what version would that be?

Reply:
https://github.com/dotnet/diagnostics/releases/tag/3.0.47001 was the first (non-preview) release of dotnet-trace. I shortened it to 3.0 but you could include the full version if that makes it clearer.